lib/brains/forms.py

```diff
@@ -61,13 +65,14 @@ class SubscriptionForm(forms.Form):
         queryset=BraintreePaymentMethod.objects.filter())
     plan = forms.CharField(max_length=255)
 
-    def clean(self):
+    def clean_plan(self):
         data = self.cleaned_data
 
         try:
             obj = SellerProduct.objects.get(external_id=data['plan'])
         except ObjectDoesNotExist:
-            raise forms.ValidationError('Seller product does not exist.')
+            raise forms.ValidationError(
+                'Seller product does not exist.', code='does_not_exist')
 
         self.seller_product = obj
         return data
```

Comment:
since you switched to ``clean_<field>`` you need to return the plan value, no? From the [docs](https://docs.djangoproject.com/en/1.7/ref/forms/validation/): "This method should return the cleaned value obtained from cleaned_data..."